roles/contiv_network/tasks/ovs_cleanup.yml

```diff
@@ -28,6 +28,7 @@
 - name: deny openvswitch_t type in selinux
   shell: >
       semanage permissive -d openvswitch_t
+  when: (ansible_os_family == "RedHat")
```

Comment:
@gaurav-dalvi sorry to nit pick. I think we can get rid of this conditional as well.